src/actions/__tests__/verify-action-store-consistency.test.js

```diff
@@ -13,11 +13,12 @@ import { jsLanguageDefinition } from '../../state-schemas/mirrored-state-schema'
 // note that updateCellProperties is not not tested, b/c this can introduce
 // arbitrary props with arbitrary values into cells
 
-const mockUserData = {
-  accessToken: 'accessToken',
-  name: 'name',
-  avatar: 'avatar',
-}
+// NOTE: muting this for the new login workflow.
+// const mockUserData = {
+//   accessToken: 'accessToken',
+//   name: 'name',
+//   avatar: 'avatar',
+// }
```

Comment:
I would vote for just removing this code, instead of just commenting it out. We can always use version control to bring it back if needed.